docs/options.md

```diff
@@ -54,4 +54,5 @@ Additional options for this plugin are:
 | `convertOptions` | array | `[]` | List of string options to pass to the convert engine. |
 | `convertAuto` | boolean | `true` | By default the converter automatically starts once recording completed. Use `false` to disable this behavior, allowing you to start the converter manually instead. |
 | `hotKeys` | boolean or function | `false` | Enable [keyboard hotkeys](hotkeys.md). Disabled by default. |
+| `countdown` | array | `[]` | List of steps to display as a countdown after clicking on the record button (before the recording starts). Each step has a value (string) to display and time interval (in milliseconds). For example `[{time:500,value:'3'},{time:500,value:'2'},{time:500,value:'1'}]` will display "3" for 0.5 sec, then "2" for 0.5 sec, then "1" for 0.5 sec and after that start the recording. |
```

Comment:
use proper spacing in example, e.g. `[{time: 500, value: '3'}, {time: 500, value: '2'}, {time: 500, value: '1'}]`